dashboard/config/routes.rb

```diff
@@ -221,7 +221,11 @@
       ProjectsController::STANDALONE_PROJECTS.each do |key, _|
         get "/#{key}", to: 'projects#load', key: key.to_s, as: "#{key}_project"
         get "/#{key}/new", to: 'projects#create_new', key: key.to_s, as: "#{key}_project_create_new"
-        get "/#{key}/:channel_id", to: 'projects#show', key: key.to_s, as: "#{key}_project_share", share: true
+        if key == 'weblab'
+          get "/#{key}/:channel_id", constraints: {host: CDO.dashboard_hostname}, to: redirect("//#{CDO.site_host('codeprojects.org')}/%{channel_id}/")
```

Comment:
Should we add a comment here about why weblab is treated differently?  (Up to you.)

Reply:
I added a comment that weblab utilizes codeprojects, though didn't go into all the details of why